File: samples/standard/fuzzy/fuzzy.py
import sys
import antlr3
from FuzzyJava import FuzzyJava

stream = antlr3.StringStream(open(sys.argv[1]).read())
lexer = FuzzyJava(stream)

while True:
    token = lexer.nextToken()
    if token.type == antlr3.EOF:
        break

File: samples/standard/fuzzy/fuzzy.rb
#!/usr/bin/ruby
# encoding: utf-8
$:.unshift( File.dirname( __FILE__ ) )
require 'FuzzyJava'

for file in ARGV
  input = ANTLR3::FileStream.new( file )
  FuzzyJava::Lexer.new( input ).exhaust
end
